### ISSUE TITLE
[4.0] Global Configuration is behaving erratically

### ISSUE THREAD

**Issue by ghazal** [joomla/joomla-cms#24678]
For a while, it's been difficult to save my Settings in Global Configuration.
For example:
Default List Limit
Session Lifetime 
Error Reporting
I have to click on Save, or Save and Close several times to get these Settings saved properly.

### System information (as much as possible)
 PHP 7.2.5
Last build (Monday, 22 April)
Installed via CLI, Composer and NPM included

**Comment by alikon** [comment]
can you specify if is 3.x or 4.x please

**Comment by SharkyKZ** [comment]
If you have OPCache enabled, see if #21949 helps. It hasn't been merged into 4.0 yet.

**Comment by ghost** [comment]
I get the reported Issue also sometimes but guessed it is a Sideeffect of Updating Nightly on Nightly (no Cache, using Defaultvalues).
### System information
Setting | Value
-- | --
PHP Built On | Linux lamp122.cloudaccess.net 3.10.0-962.3.2.lve1.5.24.7.el6h.x86_64 nr.1 SMP Mon Dec 17 12:02:35 EST 2018 x86_64
Database Type | mysql
Database Version | 5.7.24-cll-lve
Database Collation | utf8_general_ci
Database Connection Collation | utf8_general_ci
PHP Version | 7.1.26
Web Server | Apache
WebServer to PHP Interface | cgi-fcgi
Joomla! Version | Latest Nightly Build Update from Joomla! 4.0.0-alpha8-dev Development [ Amani ] 9-March-2019 13:41 GMT
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:66.0) Gecko/20100101 Firefox/66.0

**Comment by ghazal** [comment]
@franz-wohlkoenig 
generally, I don't update via Nightly. I install a new batch via CLI  if I want to write about a new issue.

@SharkyKZ 
Re PR #21949
It looks like this works.
```
		if (function_exists('opcache_invalidate'))
		{
			opcache_invalidate($file);
		}
		
```

I inserted it manually because since this PR, the name and path of the file had changed and became:
/administrator/components/com_config/Model/ApplicationModel.php
instead of:
administrator/components/com_config/model/application.php


So I guess a new PR should be made.

**Comment by ghost** [comment]
@ghazal i'm waiting on Beta where Night-Updates on Nightly should work.

**Comment by ghost** [comment]
Closed as having Pull Request.
